docs/examples.py

```diff
@@ -122,7 +122,7 @@ def Wrapper():
     def PrintView():
         text, set_text = idom.hooks.use_state(print_buffer.getvalue())
         print_buffer.set_callback(set_text)
-        return idom.html.pre(text, class_name="printout") if text else idom.html.div()
+        return idom.html.pre({"class": "printout"}, text) if text else idom.html.div()
```

Comment:
`"class"` needs a revert to `"class_name"`. Also confirm throughout the docs we aren't missing any of those situations.

Reply:
```suggestion
        return idom.html.pre({"class_name": "printout"}, text) if text else idom.html.div()
```